### DISCUSSION TITLE
Windows default beep playing when Enter key or certain command keys are pressed

### DISCUSSION THREAD

**Discussion by Huhenyo** [scandum/tintin#237] [General]
I'm using tt++.exe to open a console that I'm using for a script.  However, most every time the Enter key is pressed, the Windows Default Beep sound plays.  This is quite annoying since i hit Enter a lot.  Is there any way to suppress this short of turning the Windows Beep sound to "None"?

Thank you!

**Comment by scandum** [comment]
You could try `#bell volume 1` to turn the bell off in tintin.

It might also be possible to substitute the bell character. `#sub {~\a} {\0}`

**Reply by Huhenyo** [reply]
Thank you!  I'll give that a try.
